pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1094,10 +1120,13 @@ def CostObjectiveFunctionRule(model):
                 + model.v_C_TotalPiping
                 + model.v_C_TotalStorage
                 + model.v_C_TotalTrucking
-                + model.v_C_DisposalCapEx
-                + model.v_C_StorageCapEx
-                + +model.v_C_TreatmentCapEx
-                + model.v_C_PipelineCapEx
+                + model.p_alpha_AnnualizationRate
+                * (
+                    model.v_C_DisposalCapEx
+                    + model.v_C_StorageCapEx
+                    + model.v_C_TreatmentCapEx
+                    + model.v_C_PipelineCapEx
+                )
```

Comment:
missing annualization rate for reuse objective function

Reply:
Thanks! 

Related to PR #81: Melody merged in applicable changes in commit 4ea6a5a.